Documentation/compatibility.md

```diff
@@ -32,3 +32,4 @@ The versions of Prometheus compatible to be run with the Prometheus Operator are
 * v1.7.2
 * v1.8.0
 * v2.0.0
+* v2.2.0
```

Comment:
This is doc is generated, it needs to be adapted in the code: https://github.com/coreos/prometheus-operator/blob/eada88b66456f7e608e450afdb80870f68746dda/pkg/prometheus/statefulset.go#L74

Reply:
@brancz well spotted